osm2gtfs/core/osm_connector.py

```diff
@@ -598,7 +646,7 @@ def _find_best_name_for_unnamed_stop(self, stop):
         winner_distance = sys.maxint
         for candidate in candidates:
             if isinstance(candidate, overpy.Way):
-                lat, lon = Stop.get_center_of_nodes(
+                lat, lon = OsmConnector.get_center_of_nodes(
```

Comment:
We do you need to prepend method calls with `OsmConnector` when you are in `OsmConnector`?